_data/harnesses/kafka-connect-datagen-local/kafka.yml

```diff
@@ -0,0 +1,79 @@
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/init.adoc
+
+    - title: Get Confluent Platform
+      content:
+        - action: make_file
+          file: Dockerfile
+          render:
+            file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/make-dockerfile.adoc
+
+        - action: make_file
+          file: docker-compose.yml
+          render:
+            file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/make-docker-compose.adoc
+
+        - action: execute_async
+          file: tutorial-steps/dev/docker-compose-up.sh
+          render:
+            file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/start-compose.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/wait-for-containers.sh
+          render:
+            skip: true
+            
+    - title: Create the connector
+      content:
+        - action: execute
+          file: tutorial-steps/dev/create-connector.sh
+          stdout: tutorial-steps/dev/outputs/create-connector.log
+          render:
+            file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/create-connector.adoc
+
+        - name: give Kafka Connect chance to create the connector
+          action: sleep
+          ms: 5000
+          render:
+            skip: true
+
+        - action: execute
+          file: tutorial-steps/dev/check-connector.sh
+          stdout: tutorial-steps/dev/outputs/check-connector.log
+          render:
+            file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/check-connector.adoc
+
+        - name: give Kafka Connect further chance to get the data to the topic
+          action: sleep
+          ms: 5000
+          render:
+            skip: true
+
+    - title: Consume events from the Kafka topic
+      content:
+      - action: execute_async
+        file: tutorial-steps/dev/harness-console-consumer-keys.sh
+        stdout: tutorial-steps/dev/outputs/consume-topic.log
+        render:
+          file: tutorials/kafka-connect-datagen-local/kafka/markup/dev/consume-topic-key-value.adoc
+
+      - name: wait for consumer to read records
+        action: sleep
+        ms: 10000
+        render:
+          skip: true
+
+prod:
```

Comment:
Is it possible to leave the prod steps out entirely as they aren't relevant for this tutorial?   The Cleanup Docker step seems like a "Dev" step IMO

Reply:
It's setup this way to allow users to do this: https://github.com/confluentinc/kafka-tutorials#run-a-tutorial

```
make SEQUENCE="dev, test"
```

Reply:
FYC add a quick note to `clean-up.adoc` that the prod step is only for clean-up.  Or maybe we can find a way to have some sort of flag to set that is skipped when running `make SEQUENCE="dev, test"`

Reply:
In lieu of changing the framework to support a flag to skip cleanup, I went the "easy" route of adding the note to `clean-up.adoc`